### PR TITLE
testing with values for air drag and water head

### PR DESCRIPTION
**Pull request by bqpd**
this goes infeasible if you reduce the head enough to significantly affect the wattage output...maybe drag is going to infinity and A_seg[0] to 0???

### CODE REVIEW

File: heatexchanger/sol.txt [hunk 1]
```diff
@@ -1,132 +1,132 @@
...
         | Layer
-  D_air : 0.02093                                         [N]        total air drag
-D_water : 0.01812                                         [N]        total water drag
-      Q : 117.9                                           [W]        heat transferred from air to liquid
- V_mtrl : 79.02                                           [cm**3]    volume of material
-  V_tot : 100                                             [cm**3]    total volume
+  D_air : 0.1                                             [N]        total air drag
+      Q : 266.4                                           [W]        heat transferred from air to liquid
+      V : 100                                             [cm**3]    total volume
+ V_mtrl : 15.33                                           [cm**3]    volume of material
```
Comment (bqpd) [inline heatexchanger/sol.txt]: it's a bit unintuitive that the low-drag one is the one that's 79% material...we seem in general to not be very sensitive to material quantity?

File: heatexchanger/sol.txt [hunk 2]
```diff
@@ -1,124 +1,123 @@
...
 --------------
         | Layer
-  D_air : 0.02093                                         [N]        total air drag
-D_water : 0.01812                                         [N]        total water drag
-      Q : 117.9                                           [W]        heat transferred from air to liquid
+  D_air : 0.1                                             [N]        total air drag
+      Q : 266.4                                           [W]        heat transferred from air to liquid
       V : 100                                             [cm**3]    total volume
- V_mtrl : 78.24                                           [cm**3]    volume of material
+ V_mtrl : 15.33                                           [cm**3]    volume of material
```
Comment (bqpd) [inline heatexchanger/sol.txt:12]: it's a bit unintuitive that the low-drag one is the one that's 79% material...we seem in general to not be very sensitive to material quantity?

**Comment by 1ozturkbe** [comment]
are we keeping this PR @bqpd? I would like to start wrapping things up, and this seems like an old PR that will never be merged.

**Comment by bqpd** [comment]
Yup, good call!

**Pull request closed without merging**